financial-connections/src/main/java/com/stripe/android/financialconnections/exception/UnclassifiedError.kt

```diff
@@ -0,0 +1,9 @@
+package com.stripe.android.financialconnections.exception
+
+internal class UnclassifiedError(
```

Comment:
This is used so we can properly log unexpected states in this flow (see `handleUnsupportedRepairAction()` and `handleUnsupportedNextPane()`).